### PR TITLE
Adding the dev-launcher

### PR DESCRIPTION
**Pull request by 2roy999**

### CODE REVIEW

**Comment by 2roy999** [comment]
With general upgrades of the pacakges and more

File: client/script.js [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
-
-}) ({})
+import '@first-lego-league/user-interface/current/assets/css/app.css'
+import './stylesheet.css'
+
+const PASSWORD_FIELD_ID = 'password-field'
+const PASSWORD_BUTTONS_CLASS = 'password-button'
+
+function onReady () {
+  const showPassword = () => {
```
Comment (idanstark42) [inline client/script.js]: Call this `togglePasswordVisibility` and not `showPassword`. It not only shows the password.

File: package.json [hunk 2]
```diff
@@ -4,22 +4,30 @@
   "description": "Identity Provider using jwt-redirection",
   "main": "server.js",
   "scripts": {
-    "start": "set DEV=true&& start mhub-server&& node server.js",
-    "lint": "eslint \"*.js\"",
-    "lint:fix": "eslint --fix \"*.js\""
+    "clean": "rimraf public",
```
Comment (idanstark42) [inline package.json]: Just for my own edification: what does this do?

Comment (2roy999) [reply package.json]: deletes the directory `public`. the name kind of comes from the linux command `rm -rf`

**Pull request merged**